tests/Integration/Behaviour/Features/Scenario/Order/update_order_shipping_details.feature

```diff
@@ -0,0 +1,19 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --name 'Order shipping'
+@reset-database-before-feature
+Feature: Order shipping details from Back Office
+  PrestaShop allows to Update shipping details of the chosen order
+  As a BO user
+  I need to be able to go to order view, select Shipping and Edit->Update shipping details
```

Comment:
this line is written from user's perspective, should be from functional perspective

Reply:
for example: `I need to be able to update shipping details of an order`